### ISSUE TITLE
Release Twisted 13.2

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6795]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#6795|
|Type|task|
|Created|2013-10-21 13:39:17Z|
|Branch|https://github.com/twisted/twisted/tree/releases/release-13.2.0-6795|

It's about that time of year when we get our release management groove on.

<details><summary>Searchable metadata</summary>

```
trac-id__6795 6795
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__ 
branch__branches_releases_release_13_2_0_6795 branches/releases/release-13.2.0-6795
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1382362757000000 1382362757000000
changetime__1383940173000000 1383940173000000
version__None None
owner__hawkowl hawkowl
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl removed owner|
|-|-|

Okay, I think we should be good to go.

Please review!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @hawkowl|
|-|-|

Thanks.  This looks fine.  Please proceed!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl set status to `closed`|
|-|-|

(In [40598]) Merge releases/release-13.2.0-6795: Release Twisted 13.2.0

Author: hawkowl
Reviewer: exarkun
Fixes: [#6795](https://github.com/twisted/twisted/issues/6795)

Release Twisted 13.2.0
